src/adapter.rs

```diff
@@ -81,19 +86,25 @@ unsafe fn get_adapter(adapter_addresses_ptr: PIP_ADAPTER_ADDRESSES) -> Result<Ad
 
     let description = WideCString::from_ptr_str(adapter_addresses.Description).to_string()?;
     let friendly_name = WideCString::from_ptr_str(adapter_addresses.FriendlyName).to_string()?;
+    let physical_address = if adapter_addresses.PhysicalAddressLength == 0 {
+        None
+    } else {
+        Some(adapter_addresses.PhysicalAddress[..adapter_addresses.PhysicalAddressLength as usize].to_vec())
+    };
     Ok(Adapter {
         adapter_name: adapter_name,
         ip_addresses: unicast_addresses,
         dns_servers: dns_servers,
         description: description,
         friendly_name: friendly_name,
+        physical_address: physical_address,
     })
 }
 
 unsafe fn socket_address_to_ipaddr(socket_address: &SOCKET_ADDRESS) -> IpAddr {
     let sockaddr = socket2::SockAddr::from_raw_parts(std::mem::transmute(socket_address.lpSockaddr), socket_address.iSockaddrLength);
 
-    // Could be either ipv4 or ipv6
+// Could be either ipv4 or ipv6
```

Comment:
I guess it's a mistake? The indentation was fine.

Reply:
Fixed